.snyk

```diff
@@ -0,0 +1,125 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-ORGAPACHETOMCAT-460328:
+    - '*':
```

Comment:
❓ What is controlled with this `'*'`?

Reply:
This is set by the `--path` flag. `*` is the default value which is used when no path is given to the `ignore` command. It's used to narrow done the scope of ignore rule.
More information [here](https://docs.snyk.io/snyk-cli/commands/ignore#path-less-than-path_to_resource-greater-than).